uisdk/src/main/java/com/karhoo/uisdk/screen/rides/detail/RideDetailView.kt

```diff
@@ -219,11 +209,11 @@ class RideDetailView @JvmOverloads constructor(
         if (cancellationDialog?.isShowing == true) {
             cancellationDialog?.dismiss()
         }
-        cancelRideButton.visibility = View.GONE
+        contactOptionsWidget.disableCancelButton()
     }
 
     override fun hideContactFleetButton() {
-        contactFleetButton.visibility = View.GONE
+        contactOptionsWidget.enableCancelButton()
```

Comment:
This logic does not seem to be working for TripDetails screen. When the Trip is completed, now we display 3 buttons and the Contact Fleet button is redundant. You need to wait for the trip to complete

Reply:
Do you mean Trip Details or Ride Details?

Reply:
Ride details

Reply:
Okay. Has been fixed in the UI PR. Will add a test there to confirm.